ui/src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -255,7 +259,7 @@ export default defineComponent({
   created() {
     // In absence of condition, emit directly to the parent the default value
     if (isEqual(this.value, DEFAULT_FILTER)) {
-      this.$emit('input', DEFAULT_FILTER);
+      this.$emit('input', { ...DEFAULT_FILTER });
```

Comment:
Une raison de pas faire this.value plutôt? Ca éviterait de faire deux spread operator et la value est déjà isolée non :thinking: ?

Reply:
Tu as raison, mieux vaut ne pas changer l'existant - ce n'est pas le but de cette PR. Je remet!